### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by a5assoapga**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # pull-requests
 
-Hello !!
+Hello, test !!
```
Comment (ovpfvq0ghh) [inline README.md]:
```suggestion
Hello, world !!
```

**Review by ovpfvq0ghh — CHANGES_REQUESTED** [review]
Change to *world*

**Review by ovpfvq0ghh — APPROVED** [review]

**Review by ovpfvq0ghh — APPROVED** [review]
LGTM

**Comment by roddavinod99** [comment]
Oh! Are you new to GitHub?. Good luck. Practice to visualize how the GitHub works so that you can learn easily.

**Pull request merged**